### PR TITLE
Move settings to dynaconf

### PR DESCRIPTION
**Pull request by PChild**
Settings for TBA, Elasticsearch, Google, Discord, and draft specific stuff all pulled out of specific files and into a settings file. 

Opted for YAML over TOML for the settings file, currently accessing settings as attributes instead of as dictionary items, can change format if anyone feels strongly.

Readme is updated with instructions for setting up `config/.secrets.yaml` instead of using `keys.py`.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -76,7 +70,7 @@ def get_draft(key: str) -> Draft:
         reg_close_time_dt = datetime.datetime.strptime(reg_close, '%Y-%m-%d %H:%M %z')
         reg_close_time_dt += datetime.timedelta(hours=12)
     except ValueError:
-        embed = discord.Embed(color=0xe8850d, title="ERROR in `init`")
+        embed = discord.Embed(color=settings.DISCORD.TITLE_COLOR, title="ERROR in `init`")
```
Comment (tervay) [inline bot.py:73]: Check the cases on this. Pretty sure it's case sensitive and throwing an error. Not sure though

```
(env) λ python bot.py
I am running as [Justin] SLFF Draft Bot
Ignoring exception in command test:
Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 423, in __getitem__
    value = super(Box, self).__getitem__(item)
KeyError: 'TITLE_COLOR'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 17, in __getitem__
    return super(DynaBox, self).__getitem__(item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 430, in __getitem__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'TITLE_COLOR'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 423, in __getitem__
    value = super(Box, self).__getitem__(item)
KeyError: 'title_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 505, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 20, in __getitem__
    return super(DynaBox, self).__getitem__(n_item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 430, in __getitem__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'title_color'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 507, in __getattr__
    value = object.__getattribute__(self, item)
AttributeError: 'DynaBox' object has no attribute 'TITLE_COLOR'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 10, in __getattr__
    return super(DynaBox, self).__getattr__(item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 524, in __getattr__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'DynaBox' object has no attribute 'TITLE_COLOR'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 423, in __getitem__
    value = super(Box, self).__getitem__(item)
KeyError: 'title_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 17, in __getitem__
    return super(DynaBox, self).__getitem__(item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 430, in __getitem__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'title_color'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 423, in __getitem__
    value = super(Box, self).__getitem__(item)
KeyError: 'TITLE_COLOR'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 505, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 20, in __getitem__
    return super(DynaBox, self).__getitem__(n_item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 430, in __getitem__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'TITLE_COLOR'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 507, in __getattr__
    value = object.__getattribute__(self, item)
AttributeError: 'DynaBox' object has no attribute 'title_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 63, in wrapped
    ret = await coro(*args, **kwargs)
  File "bot.py", line 58, in test
    await init._callback(ctx, "MidKnight Mayhem", "2019-06-01", "2:30", "4:30")
  File "bot.py", line 108, in init
    embed = discord.Embed(color=settings.DISCORD.TITLE_COLOR, title=title_msg)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\dynaconf\utils\boxing.py", line 13, in __getattr__
    return super(DynaBox, self).__getattr__(n_item, *args, **kwargs)
  File "E:\Code\SLFF-Drafter\env\Scripts\box.py", line 524, in __getattr__
    raise BoxKeyError(str(err))
box.BoxKeyError: "'DynaBox' object has no attribute 'title_color'"

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\bot.py", line 860, in invoke
    await ctx.command.invoke(ctx)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 698, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 72, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: BoxKeyError: "'DynaBox' object has no attribute 'title_color'"

**Review by tervay — CHANGES_REQUESTED** [review]

File: config/settings.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+default:
+    tba:
+        api_key: 9qTowkNEd3IarS0iDGB40d6Gqi4YJDlosHiLeLypQ3XfAEFeBp0bIYSqcBqB3fHb
+
+    discord:
+        bot_user_id: 573557278695882762
+        register_emoji: \U0001F44D
```
Comment (tervay) [inline config/settings.yaml]:
```
(env) λ python bot.py
I am running as [Justin] SLFF Draft Bot
Ignoring exception in command test:
Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 63, in wrapped
    ret = await coro(*args, **kwargs)
  File "bot.py", line 58, in test
    await init._callback(ctx, "MidKnight Mayhem", "2019-06-01", "2:30", "4:30")
  File "bot.py", line 119, in init
    await sent.add_reaction(settings.DISCORD.REGISTER_EMOJI)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\message.py", line 716, in add_reaction
    await self._state.http.add_reaction(self.id, self.channel.id, emoji)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\http.py", line 222, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: BAD REQUEST (status code: 400): Unknown Emoji

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\bot.py", line 860, in invoke
    await ctx.command.invoke(ctx)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 698, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "E:\Code\SLFF-Drafter\env\lib\site-packages\discord\ext\commands\core.py", line 72, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: BAD REQUEST (status code: 400): Unknown Emoji
```

Occurs when running `.test`

**Review by tervay — CHANGES_REQUESTED** [review]

File: config/settings.yaml (hunk 2 above)
Comment (PChild) [reply config/settings.yaml]: Yeah, working on this. I hate unicode.

**Review by tervay — APPROVED** [review]
lgtm

**Pull request merged**